src/main/scala/sparkz/core/network/DeliveryTracker.scala

```diff
@@ -250,4 +261,18 @@ class DeliveryTracker(system: ActorSystem,
         log.warn("Unexpected error", e)
         Failure(e)
     }
+
+  private def updateProcessingTime(startTime: Long): Unit = {
+    if (slowModeFeatureFlag) {
+      val elapsedMs: Long = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startTime)
+      averageProcessingTimeMs = (averageProcessingTimeMs * 0.9).toLong + (elapsedMs * 0.1).toLong
```

Comment:
does this 0.1 comes from a well known formula?  looks like it will influence the "speed" at which the filter will be activated/deactivated right? let's consider to make it parametric in settings file to allow tuning

Reply:
This is inspired by the GETH solution when they update peers throughput by 0.1x of each new measurement + 0.9x of existing. So that a single long-running process won't immediately lead to a slow mode activation. I will add a configuration for this